js/weather-api/api.js

```diff
@@ -0,0 +1,26 @@
+const setWaeather = (location) =>{
+    const queryParams = new URLSearchParams({
+        appid: "4b5774e9f3d2a07b84f0f2f88e486224", 
+        q: location,
+        lang:"ja" 
+    });
+    const myRequest = new Request("https://api.openweathermap.org/data/2.5/weather?" + queryParams)
```

Comment:
[must]

```suggestion
    const request = new Request("https://api.openweathermap.org/data/2.5/weather?" + queryParams)
```

クラス名が`Request`のため、それと同じにならないように`my`をつけてくれたのでしょうか？
クラス名と変数名がおなじになることはよくあるので、それ自体は問題ありません。

慣例的に`my`など利用することは少ないのですが、もし理由があったら教えてほしいです。
特に理由がなければ`request`としてしまうのが良いと思います

Reply:
[info]
`URLSearchParams` を利用しているので、URL文字列の作成部分も`URL`を利用すると良いと思います。

`+` による文字列結合と結果は一緒ですが、下記の理由から 
- `+` を利用するよりもバグを埋め込む可能性が減る（+を使うと、半角が入ったりして予期せぬ文字列になることがあります）
- ↑↑↑と同じ理由ですが、可読性がよくなる 

参考
- [searchをURLSearchParamsで置換する](https://qiita.com/ovrmrw/items/1c1564481c4ca9cc4351#search%E3%82%92urlsearchparams%E3%81%A7%E7%BD%AE%E6%8F%9B%E3%81%99%E3%82%8B)